src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -7,6 +7,10 @@ public interface CinemaHallService {
     CinemaHall add(CinemaHall cinemaHall);
 
     CinemaHall get(Long id);
-    
+
     List<CinemaHall> getAll();
+
+    Boolean update(CinemaHall cinemaHall);
```

Comment:
Consider changing the return type of the `update` method to `CinemaHall` or `void` to better reflect its operation. Returning a Boolean might not provide enough information about the update process.